src/LiveComponent/src/Controller/BatchActionController.php

```diff
@@ -37,7 +37,7 @@ public function __invoke(Request $request, MountedComponent $_mounted_component,
                 '_controller' => [$serviceId, $name],
                 '_component_action_args' => $action['args'] ?? [],
                 '_mounted_component' => $_mounted_component,
-                '_route' => 'ux_live_component',
+                '_live_component' => $serviceId,
```

Comment:
Fixed, this was the culprit (since the `_route` doesn't trigger the subscriber anymore).